src/test-run/commands/index.js

```diff
@@ -98,6 +100,13 @@ function navigateToUrlArgument (name, val) {
         throw new ActionUnsupportedUrlProtocolError(name, protocol[0]);
 }
 
+function resizeWindowDeviceArgument (name, val) {
+    nonEmptyStringArgument(name, val);
+
+    if (!isValidDeviceName(val.trim()))
```

Comment:
I suppose there is no need for trimming. Due to [sources](https://github.com/DevExpress/testcafe-browser-natives/blob/master/src/utils/get-device-size.js#L8):
`deviceName = deviceName.toLowerCase().split(' ').join('');`

Leading and trailing spaces will be removed.